File: src/net/eduard/api/command/api/ApiUnloadCommand.java

package net.eduard.api.command.api;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class ApiUnloadCommand extends CommandManager {

	public ApiUnloadCommand() {
		super("unload","descarregar");
		setUsage("/api unload <plugin>");
		setDescription("Descarrega um plugin carregado no servidor");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command,
			String label, String[] args) {
		sender.sendMessage("§cNão implementado");
		
		return true;
	}

}

File: src/main/java/net/eduard/api/command/api/ApiUnloadCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ApiUnloadCommand : CommandManager("unload", "descarregar") {
    override fun onCommand(
        sender: CommandSender, command: Command,
        label: String, args: Array<String>
    ): Boolean {
        sender.sendMessage("§bEduardAPI §cNão implementado")
        return true
    }

    init {
        usage = "/api unload <plugin>"
        description = "Descarrega um plugin carregado no servidor"
    }
}